noxfile.py

```diff
@@ -16,13 +16,13 @@
 
 
 #: Default to reusing any pre-existing nox environments.
-nox.options.reuse_existing_virtualenvs = True
+nox.options.reuse_existing_virtualenvs = False
```

Comment:
Do we want to keep this? Will it break this PR's CI to change it back to `True`?